prow/jobs/test-infra/prow-periodics.yaml

```diff
@@ -96,6 +96,33 @@ periodics: # runs on schedule
           - name: config
             configMap:
               name: label-config
+    - name: eventing-tools-autobump
+      annotations:
+        testgrid-create-test-group: "false"
+      labels:
+        prow.k8s.io/pubsub.project: "sap-kyma-prow"
+        prow.k8s.io/pubsub.runID: "eventing-tools-autobump"
+        prow.k8s.io/pubsub.topic: "prowjobs"
+        preset-autobump-bot-github-token: "true"
+      cron: "30 * * * 1-5"
+      skip_report: false
+      decorate: true
+      cluster: trusted-workload
+      extra_refs:
```

Comment:
I think you need eventing-tools repo defined in extra_refs too.
@Ressetkk could you please confirm if I'm correct?

Reply:
i moved it to the eventing-tools repository